localization/gyro_odometer/src/gyro_odometer_core.cpp

```diff
@@ -105,25 +107,28 @@ void GyroOdometer::callbackImu(const sensor_msgs::msg::Imu::ConstSharedPtr imu_m
   twist.header.stamp = imu_msg_ptr_->header.stamp;
   twist.header.frame_id = output_frame_;
   twist.twist.linear = twist_with_cov_msg_ptr_->twist.twist.linear;
-  twist.twist.angular.z = transformed_angular_velocity.vector.z;  // TODO(YamatoAndo) yaw_rate only
+  twist.twist.angular.x = transformed_angular_velocity.vector.x;
+  twist.twist.angular.y = transformed_angular_velocity.vector.y;
+  twist.twist.angular.z = transformed_angular_velocity.vector.z;
   twist_pub_->publish(twist);
 
   geometry_msgs::msg::TwistWithCovarianceStamped twist_with_covariance;
   twist_with_covariance.header.stamp = imu_msg_ptr_->header.stamp;
   twist_with_covariance.header.frame_id = output_frame_;
   twist_with_covariance.twist.twist.linear = twist_with_cov_msg_ptr_->twist.twist.linear;
-  twist_with_covariance.twist.twist.angular.z =
-    transformed_angular_velocity.vector.z;  // TODO(YamatoAndo) yaw_rate only
+  twist_with_covariance.twist.twist.angular.x = transformed_angular_velocity.vector.x;
+  twist_with_covariance.twist.twist.angular.y = transformed_angular_velocity.vector.y;
+  twist_with_covariance.twist.twist.angular.z = transformed_angular_velocity.vector.z;
 
   // NOTE
-  // linear.y, linear.z, angular.x, and angular.y are not measured values.
+  // linear.y and linear.z are not measured values.
   // Therefore, they should be assigned large variance values.
   twist_with_covariance.twist.covariance[0] = twist_with_cov_msg_ptr_->twist.covariance[0];
   twist_with_covariance.twist.covariance[7] = 10000.0;
   twist_with_covariance.twist.covariance[14] = 10000.0;
-  twist_with_covariance.twist.covariance[21] = 10000.0;
-  twist_with_covariance.twist.covariance[28] = 10000.0;
-  twist_with_covariance.twist.covariance[35] = imu_msg_ptr_->angular_velocity_covariance[8];
+  twist_with_covariance.twist.covariance[21] = imu_msg_ptr->angular_velocity_covariance[0];
```

Comment:
```suggestion
  twist_with_covariance.twist.covariance[21] = imu_msg_ptr_->angular_velocity_covariance[0];
```

Reply:
Why do you suggest adding the trailing underscore?

Reply:
The underscores are attached in other parts of the code, so I think it is better to do the same.